.travis.yml

```diff
@@ -24,94 +27,97 @@ matrix:
+    #   os: linux
+    #   dist: bionic
+    #   services:
+    #     - docker
+    #   env:
+    #     - DOCKER_IMAGE="debian:sid"     DOCKER_DEPS_CMD="apt update && apt install -y libglib2.0-0 procps gcc git curl"
+    # - name: "Debian Testing"
+    #   if: branch = release or type = cron
+    #   os: linux
+    #   dist: bionic
+    #   services:
+    #     - docker
+    #   env:
+    #     - DOCKER_IMAGE="debian:testing" DOCKER_DEPS_CMD="apt update && apt install -y libglib2.0-0 procps gcc git curl"
+    # - name: "Debian 10"
+    #   if: branch = release or type = cron
+    #   os: linux
+    #   dist: bionic # kernel ~= Debian:10
+    #   services:
+    #     - docker
+    #   env:
+    #     - DOCKER_IMAGE="debian:10"      DOCKER_DEPS_CMD="apt update && apt install -y libglib2.0-0 procps gcc git curl"
+    # - name: "Debian 9"
+    #   if: branch = release or type = cron
+    #   os: linux
+    #   dist: xenial # kernel ~= Debian:9
+    #   services:
+    #     - docker
+    #   env:
+    #     - DOCKER_IMAGE="debian:9"       DOCKER_DEPS_CMD="apt update && apt install -y libglib2.0-0 procps gcc git curl"
+    # - name: "Debian 8"
+    #   if: branch = release or type = cron
+    #   os: linux
+    #   dist: xenial # kernel ~= Debian:8
+    #   services:
+    #     - docker
+    #   env:
+    #     - DOCKER_IMAGE="debian:8"       DOCKER_DEPS_CMD="apt update && apt install -y libglib2.0-0 procps gcc git curl"
+    # - name: "CentOS 8"
+    #   # runs on all branches
+    #   os: linux
+    #   dist: bionic # kernel ~= Centos8
+    #   services:
+    #     - docker
+    #   env:
+    #     - DOCKER_IMAGE="centos:8" DOCKER_DEPS_CMD="yum install -y which gcc git curl"
+    # - name: "CentOS 7"
+    #   if: branch = release or type = cron
+    #   os: linux
+    #   dist: xenial # kernel ~= Centos7
+    #   services:
+    #     - docker
+    #   env:
+    #     - DOCKER_IMAGE="centos:7" DOCKER_DEPS_CMD="yum install -y which gcc git curl"
+    # # The rest of the OSes can use Travis's built-in images:
+    # - name: "Ubuntu 18.04 (Bionic Beaver)"
+    #   # runs on all branches
+    #   os: linux  # list of Linux env: https://docs.travis-ci.com/user/reference/overview/
+    #   dist: bionic
+    # - os: linux
+    #   name: "Ubuntu 16.04 (Xenial)"
+    #   if: branch = release or type = cron
+    #   dist: xenial
+    # - os: linux
+    #   if: branch = release or type = cron
+    #   name: "Ubuntu 14.04 (Trusty)"
+    #   dist: trusty
+    # - name: "OSX 10.14 (Mojave)"
+    #   # runs on all branches
+    #   os: osx  # list of OSX env: https://docs.travis-ci.com/user/reference/osx/
+    #   osx_image: xcode11.2
+    # - name: "OSX 10.13 (High Sierra)"
+    #   if: branch = release or type = cron
+    #   os: osx
+    #   osx_image: xcode9.4
+    # - name: "OSX 10.12 (Sierra)"
+    #   if: branch = release or type = cron
+    #   os: osx
+    #   osx_image: xcode9.2
+    - name: "Windows 10"
+      language: bash
```

Comment:
This line isn't necessary.

https://docs.travis-ci.com/user/reference/windows/:

> Git BASH is the shell that’s used to run your build. See Git for Windows for more details.

```suggestion
```

Reply:
It is not, the windows builds only support bash and it defaults to ruby. I suppose language could be set to python but bash seemed easiest because our build script is bash.

Reply:
Alright!